Formula/flash.rb

```diff
@@ -0,0 +1,57 @@
+class Flash < Formula
+  desc "Command-line script to flash SD card images of any kind"
+  homepage "https://github.com/hypriot/flash"
+  url "https://github.com/hypriot/flash/releases/download/2.6.0/flash"
+  sha256 "38a8a5793ae5fac138eeff781c932de98977215b7bfd20e5f3e0a7f46920c6d7"
+
+  bottle :unneeded
+
+  def install
+    bin.install "flash"
+  end
+
+  test do
+    # mock required binaries...
+    ENV.prepend_path "PATH", testpath
```

Comment:
Mocking all functional parts is not a great idea. It might be better to test for failure then.

Reply:
Ok, how about ae4c5b18675e122d6f1bb60e0922ad2ff75da738 then? 🙂